### DISCUSSION TITLE
tflite doesn't exist

### DISCUSSION THREAD

**Discussion by zotman** [codeproject/CodeProject.AI-Server#233] [Q&A]
Any time i try to run any of the YOLO 8 or 5 Coral modules i get these errors. I have reinstalled and can not get them to work:

`15:26:56:objectdetection_coral_adapter.py: ERROR:root:TFLite file C:\Program Files\CodeProject\AI\modules\ObjectDetectionCoral\assets\all_segments_yolov8m_416_640px_segment_0_of_2_edgetpu.tflite doesn't exist
15:26:56:objectdetection_coral_adapter.py: WARNING:root:Model file not found: [Errno 2] No such file or directory: 'C:\\Program Files\\CodeProject\\AI\\modules\\ObjectDetectionCoral\\assets\\all_segments_yolov8m_416_640px_segment_0_of_2_edgetpu.tflite'`

`15:30:31:objectdetection_coral_adapter.py: ERROR:root:TFLite file C:\Program Files\CodeProject\AI\modules\ObjectDetectionCoral\assets\yolov5l-int8_edgetpu.tflite doesn't exist
15:30:31:objectdetection_coral_adapter.py: WARNING:root:Model file not found: [Errno 2] No such file or directory: 'C:\\Program Files\\CodeProject\\AI\\modules\\ObjectDetectionCoral\\assets\\yolov5l-int8_edgetpu.tflite'
15:30:31:objectdetection_coral_adapter.py: WARNING:root:No Coral TPUs found or able to be initialized. Using CPU.
`

Any time i try to click download the model i get the following error as well:

`15:32:38:Downloading model 'objectdetection-yolov8-medium-edgetpu.zip' to 'C:\Program Files\CodeProject\AI\downloads\modules\packages\ObjectDetectionCoral\objectdetection-yolov8-medium-edgetpu.zip'
15:32:43:Unable to download and install objectdetection-yolov8-medium-edgetpu.zip: Unable to download 'objectdetection-yolov8-medium-edgetpu.zip'. Error: Could not find a part of the path 'C:\Program Files\CodeProject\AI\downloads\modules\packages\ObjectDetectionCoral\objectdetection-yolov8-medium-edgetpu.zip'.
15:32:49:Preparing to download model 'objectdetection-yolov5-medium-edgetpu.zip' for module ObjectDetectionCoral
15:32:49:Downloading model 'objectdetection-yolov5-medium-edgetpu.zip' to 'C:\Program Files\CodeProject\AI\downloads\modules\packages\ObjectDetectionCoral\objectdetection-yolov5-medium-edgetpu.zip'
15:32:50:Unable to download and install objectdetection-yolov5-medium-edgetpu.zip: Unable to download 'objectdetection-yolov5-medium-edgetpu.zip'. Error: Response status code does not indicate success: 404 (Not Found).
`

Thanks in advance

**Comment by ChrisMaunder** [comment]
Which version of the server are you running?

**Reply by zotman** [reply]
Currently 2.9.5 but it seems to have had the same exact error on the last few versions

**Comment by hpmkruit** [comment]
Check if the folder C:\Program Files\CodeProject\AI\downloads\modules\packages\ObjectDetectionCoral\ exists. Else create it manually. Something to do with permissions. Had this issue when downloading models for Coral TPU module on version 2.9.5

**Reply by zotman** [reply]
The folder did exist and was populated with some modules, but just not named correctly.

**Comment by ChrisMaunder** [comment]
Check you can download directly: https://codeproject-ai-bunny.b-cdn.net/server/assets/models/objectdetection-yolov8-medium-edgetpu.zip

**Reply by zotman** [reply]
The files in that zip are not named the same as which the error message is giving me

**Comment by mpl1337** [comment]
Same problem here.


![image](https://github.com/user-attachments/assets/6767710f-21b5-47b6-82bd-5fa0a26fa434)

![image](https://github.com/user-attachments/assets/e877521f-b440-4aa0-b474-94aec1950976)

The file doesnt exist. Iv changed the FIlenames to get it working(It works). Is this OK?



yolov8m_416_640px_segment_0_of_2_edgetpu.tflite -> **all_segments_** yolov8m_416_640px_segment_0_of_2_edgetpu.tflite




![image](https://github.com/user-attachments/assets/9d7bf5da-2b49-46cd-b4c5-65954ae3f41a)
